src/server/scripts/poller.js

```diff
@@ -205,6 +205,12 @@ export class GitSourcePart {
       var sourceUrl = part['source'];
       var sourceBranch = part['source-branch'];
       var sourceTag = part['source-tag'];
+      const { url } = parseGitHubRepoUrl(sourceUrl);
+      if (sourceUrl.replace('.git', '') != url) {
```

Comment:
What in case of repo URL without `.git`, which I believe is also valid, such as: `http://github.com/canonical-websites/build.snapcraft.io`? We don't support those in parts?

Reply:
@bartaz `git-parse-url` rebuilds the short-url from `owner` and `name` and it just normalizes what we get from snapcraft.yaml before comparing.
I've adjusted the test to maker sure short and full GH urls are supported.

Reply:
@caldav Ah, right. I've seen `.git` in this line and thought that you are expecting it, not replacing it. But thanks for updating the test.